src/index.ts

```diff
@@ -87,6 +91,6 @@ await new Listr(listrTasks, {
     removeEmptyLines: false,
     formatOutput: 'wrap',
   },
-  fallbackRenderer: 'verbose',
```

Comment:
this is probably a merge mistake

Reply:
https://github.com/upleveled/preflight/blob/58f22fa912f16a6bb5ae8e43bc338b72ed424199/src/index.ts#L94